### PR TITLE
Fix MISSION_ITEM_INT bugs

### PR DESCRIPTION
**Pull request by DonLakeFlyer**
Fix problem with incorrect futzing of x/y params.

### CODE REVIEW

File: src/MissionManager/PlanManager.cc [hunk 1]
```diff
@@ -394,8 +394,8 @@ void PlanManager::_handleMissionItem(const mavlink_message_t& message, bool miss
         param2 =        missionItem.param2;
         param3 =        missionItem.param3;
         param4 =        missionItem.param4;
-        param5 =        (double)missionItem.x / qPow(10.0, 7.0);
-        param6 =        (double)missionItem.y / qPow(10.0, 7.0);
```
Comment (FlavioTonelli) [inline src/MissionManager/PlanManager.cc:398]: @DonLakeFlyer looks like you used two time missionItem.x

Comment (DonLakeFlyer) [reply src/MissionManager/PlanManager.cc:398]: Ah crap, thanks

Comment (DonLakeFlyer) [reply src/MissionManager/PlanManager.cc:398]: I need to switch the unit tests to MISSION_ITEM_INT so they will catch this

Comment (DonLakeFlyer) [reply src/MissionManager/PlanManager.cc:398]: @FlavioTonelli Thanks so much for catching this

Comment (FlavioTonelli) [reply src/MissionManager/PlanManager.cc:398]: Just lucky that I scrolled into this pull request ;D

**Pull request merged**